i18n/de.json

```diff
@@ -483,6 +501,7 @@
       "couldNotSave": "Der Sketch konnte nicht gesichert werden. Bitte kopiere deine ungesicherte Arbeit in deinen bevorzugten Texteditor und starte die IDE neu.",
       "daemonOffline": "CLI Daemon Offline",
       "offline": "Offline",
+      "offlineText": "Offline",
       "quitTitle": "Sind Sie sicher, dass das Fenster schließen möchten?"
```

Comment:
```suggestion
      "quitTitle": "Sind Sie sicher, dass Sie das Fenster schließen möchten?"
```

Reply:
Hi @Hans-Arno. Thanks for your interest in improving the translations! Please submit your corrections via the Transifex project:

https://explore.transifex.com/arduino-1/ide2/

We can't accept any here because this is only an automated sync from the Transifex content so any changes made directly here would only be overwritten on the next sync.

There is additional information about contributing to the Arduino IDE translations here:

https://github.com/arduino/arduino-ide/blob/main/docs/contributor-guide/translation.md#translator-guide